example/spec/ingest/ingestFromPdrSpec.js

```diff
@@ -0,0 +1,441 @@
+'use strict';
+
+/**
+ * End to end ingest from discovering a PDR
+ *
+ * Kick off discover and queue pdrs which:
+ * Discovers 1 PDR
+ * Queues that PDR
+ * Kicks off the ParsePDR workflow
+ *
+ * Parse PDR workflow:
+ * parses pdr
+ * queues a granule
+ * pdr status check
+ * This will kick off the ingest workflow
+ *
+ * Ingest worklow:
+ * runs sync granule - saves file to file staging location
+ * performs the fake processing step - generates CMR metadata
+ * Moves the file to the final location
+ * Does not post to CMR (that is in a separate test)
+ */
+
+const { Collection, Execution, Pdr } = require('@cumulus/api/models');
+const {
+  buildAndExecuteWorkflow,
+  waitForCompletedExecution,
+  LambdaStep,
+  addProviders,
+  cleanupProviders,
+  addCollections,
+  cleanupCollections,
+  api: apiTestUtils
+} = require('@cumulus/integration-tests');
+
+const {
+  loadConfig,
+  updateAndUploadTestDataToBucket,
+  deleteFolder,
+  createTimestampedTestId,
+  createTestDataPath,
+  createTestSuffix,
+  getExecutionUrl
+} = require('../helpers/testUtils');
+
+const {
+  loadFileWithUpdatedGranuleIdPathAndCollection
+} = require('../helpers/granuleUtils');
+
+const config = loadConfig();
+const lambdaStep = new LambdaStep();
+const taskName = 'DiscoverAndQueuePdrs';
+const pdrFilename = 'MOD09GQ_1granule_v3.PDR';
+
+const s3data = [
+  '@cumulus/test-data/pdrs/MOD09GQ_1granule_v3.PDR',
+  '@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf.met',
+  '@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf'
+];
+
+describe('Ingesting from PDR', () => {
+  const testId = createTimestampedTestId(config.stackName, 'IngestFromPdr');
```

Comment:
This is really dense.  Can you please break this up into groups of related statements with blank lines between the groups?  It's pretty hard to read.